chronoline/chronoline.js

```diff
@@ -394,13 +398,47 @@ function Chronoline(domElement, events, options) {
         // DRAWING
         t.floatingSet = t.paper.set();
         t.sectionLabelSet = t.paper.set();
+
+        //attach background click events function
+        t.myCanvas.onclick = function (e) {
+            //if not event or section click.
+            if (e.target.nodeName != 'circle' && e.target.nodeName != 'rect') {
+                var clickDate = new Date(t.pxToMs(e.clientX));
+                clickDate.stripTime();
+                t.backgroundClick(clickDate);
```

Comment:
seems like maybe the prototype should be `t.backgroundClick(event, clickDate)`. Seems like good form to pass along the event to any handlers?